report/src/main/kotlin/com/hartwig/actin/report/pdf/util/Tables.kt

```diff
@@ -25,13 +29,33 @@ object Tables {
         return createSingleCol().setWidth(width)
     }
 
+    fun extractTextFromCell(cell: Cell): String {
+        return cell.children
+            .filterIsInstance<Paragraph>()
+            .joinToString("\n") { paragraph ->
+                paragraph.children
+                    .filterIsInstance<Text>()
+                    .joinToString("") { it.text }
+            }
+    }
+
     fun makeWrapping(contentTable: Table, forceKeepTogether: Boolean, skipWrappingFooter: Boolean = false): Table {
-        if (contentTable.numberOfRows == 0) {
-            contentTable.addCell(Cells.createSpanningNoneEntry(contentTable))
+        val tableOnlyContainsFootNote = if (contentTable.numberOfRows == 1) {
+            val text = extractTextFromCell(contentTable.getCell(0, 0))
+            TrialFormatFunctions.FOOTNOTES.any { text.contains(it) }
+        } else false
+
+        if (contentTable.numberOfRows == 0 || tableOnlyContainsFootNote) {
+            val tableWithoutHeaders = createSingleColWithWidth(contentTable.width.value)
+            tableWithoutHeaders.addCell(Cells.createSpanningNoneEntry(contentTable))
+            for (row in 0 until contentTable.numberOfRows) {
+                tableWithoutHeaders.addCell(contentTable.getCell(row, 0).clone(true))
+            }
+            return tableWithoutHeaders.setKeepTogether(true)
```

Comment:
I am not sure what you are doing here. If a table has 0 rows, why even insert it in a new table without headers?

Could be that I simply don't understand well enough what you are trying to accomplish here?

Also, in general, you have added quite some functionality to a function that is called `makeWrapping` that has nothing to do with wrapping. That is typically not a great design!